development-apache/Makefile

```diff
@@ -0,0 +1,30 @@
+# SHELL := /bin/bash
```

Comment:
I created a make file here so that you can restart / down / up / logs the development apache server more easily

```
make -C development-apache restart
```